packages/codegen/src/generator/ErrorsGen.ts

```diff
@@ -0,0 +1,45 @@
+import { ApiGen } from './ApiGen';
+import { stringLowerFirst } from '@polkadot/util';
+import { beautifySourceCode, commentBlock, compileTemplate } from './utils';
+
+export class ErrorsGen extends ApiGen {
+  generate() {
+    const { pallets } = this.metadata;
+
+    this.typesGen.clearCache();
+    this.typesGen.typeImports.addKnownType('GenericChainErrors', 'GenericModuleError');
+
+    let defTypeOut = '';
+    for (let pallet of pallets) {
+      const error = pallet.error;
+      if (!error) {
+        continue;
+      }
+
+      const errors = this.#getErrorVariants(error);
+
+      defTypeOut += `${stringLowerFirst(pallet.name)}: {
+        ${errors.map(({ name, docs }) => `${commentBlock(docs)}${name}: GenericModuleError`).join(',\n')}
+      },`;
```

Comment:
Let's convert both pallet name and error name to `camelCase` similar to the [polkadot.js errors api](https://polkadot.js.org/docs/substrate/errors), we should use the `stringCamelCase`, not the `stringLowerFirst`.